src/views/AgmExtension.vue

```diff
@@ -321,6 +320,12 @@ export default class AgmExtension extends Mixins(CommonMixin, DateMixin,
       return
     }
 
+    // if not eligible, display dialog
+    if (!this.data.isEligible) {
+      this.notEligibleExtensionDialog = true
+      return
+    }
```

Comment:
Sample:
![image](https://github.com/bcgov/business-filings-ui/assets/10002889/dd205a4f-ab60-4a92-8f76-73bfe6148b97)

Reply:
Note that this is the "new style" of dialogs. Filings UI mostly has "old style" dialogs.